command/init.go

```diff
@@ -66,77 +66,149 @@ func (c *InitCommand) Run(args []string) int {
 var defaultJob = strings.TrimSpace(`
 # There can only be a single job definition per file. This job is named
 # "example" so it will create a job with the ID and Name "example".
+
+# The "job" stanza is the top-most configuration option in the job
```

Comment:
@jippi it literally says that in the line above this one:

>  # There can only be a single job definition per file. This job is named
>  # "example" so it will create a job with the ID and Name "example".